src/main/java/core/basesyntax/Quadrangle.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
```

Comment:
Don't begin class or method implementation with an empty line. Please remove the redundant empty line.